xt/01.2-deps.t

```diff
@@ -34,8 +34,10 @@ find(\&collect, 'lib/', 'old/bin/', 'old/lib/');
 
 push @on_disk, 'tools/starman.psgi';
 
+#TODO: Implement Template plugins <?lsmb USE plugin ?>
```

Comment:
Why are you submitting a PR that's got TODOs in them? Is it half-done?

Reply:
Our dependency module never detected Template plugins. I put the reference manually in the list and added the TODO comment to upgrade the module. I am just trying to stay focused on the initial modification, not more.